tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -67,6 +67,14 @@ class Dummy
      */
     private $alias;
 
+    /**
+     * @var string The dummy secret property, only readable/writable by specific users
+     *
+     * @ORM\Column(nullable=true)
+     * @ApiProperty(security="is_granted('ROLE_ADMIN')")
+     */
+    private $adminOnlyProperty;
```

Comment:
the same changes must be done in `tests/Fixtures/TestBundle/Document/Dummy.php` so the mongoDB suite will pass too

Reply:
Just a suggestion, but using another Entity/Document will avoid to have to modify filters tests

Maybe this could be added to the SecuredDummy instead, it will have less side impacts and will be at the same place than the security attribute from ApiResource

Reply:
@antograssiot that what I noticed as well when I started implementing behat tests. I will move  it to `SecuredDummy` instead :)